pixorama_server/.gitignore

```diff
@@ -7,3 +7,6 @@ build/
 
 # Directory created by dartdoc
 doc/api/
+
```

Comment:
The password file is needed to run the project locally. We may want to keep it for the example project. Maybe add a comment that this should normally not be version controlled.

Reply:
Fixed